src/Servers/IIS/AspNetCoreModuleV2/CommonLib/Environment.cpp

```diff
@@ -231,7 +231,7 @@ bool Environment::CheckUpToDate(const std::wstring& source, const std::filesyste
             auto sourceInnerDirectory = std::filesystem::directory_entry(path);
             if (sourceInnerDirectory.path() != directoryToIgnore)
             {
-                CheckUpToDate(destination / path.path().filename(), path.path(), extension, directoryToIgnore);
+                CheckUpToDate(path.path(), destination / path.path().filename(), extension, directoryToIgnore);
```

Comment:
If this were C#, I would definitely use named arguments.  Comments might still be worthwhile (i.e. `/*source*/` and `/*destination*/`).

Reply:
```suggestion
                CheckUpToDate(/*source*/ path.path(), /*destination*/ destination / path.path().filename(), extension, directoryToIgnore);
```